packages/db/src/lib/granule.ts

```diff
@@ -354,3 +357,39 @@ export const getGranulesByGranuleId = async (
     .where({ granule_id: granuleId });
   return records;
 };
+
+/**
+ * Change a granules' PG record and its files' PG record based on collection move
```

Comment:
Nit: this method looks like it's purpose is to create a bulk granule update method that circumvents the existing granule write logic.     That probably isn't specific to collections.